src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-containerized-function-monitoring/enable-containerized-function.mdx

```diff
@@ -0,0 +1,241 @@
+---
+title: Create and deploy New Relic Lambda image layers
+metaDescription: After enabling New Relic's Lambda containerized function, you can create a layer as an image.
+redirects:
+freshnessValidatedDate: never
+---
+
+With Lambda containerized functions, Lambda layers can be created using container images. Instead of deploying as ZIP archives, container images allow for increased flexibility and improved security. Lambda container images are more agile and can help busy developers manage an increasingly tangled stack. 
+
+This document will guide you through creating a New Relic Lambda layer as an image. First, you'll need to build a Dockerfile that includes your chosen runtime environment, installs necessary dependencies, and copies them into the expected directory structure. Once you've created the layer as an image, you'll push the image to an Amazon ECR repository. You will reference it from there when you create your Lambda layer in the AWS Management Console. 
+
+First, you need to create image layers. Once those layers are created you can either create a regular or a public ECR function. 
+
+<Steps>
+<Step>
+### Create a New Relic Lambda layer as an image [#create-lambda-layer]
```

Comment:
Customer can choose our public layers from [here](https://gallery.ecr.aws/x6n7b2o2?page=1), Or If they want's create their own ECR then they have to follow these steps from the line 16 to 77
Please explain that here.